<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta http-equiv="Content-Style-Type" content="text/css" />
  <meta name="generator" content="pandoc" />
  <title></title>
  <style type="text/css">code{white-space: pre;}</style>
  <style type="text/css">
  /*
   * Copyright 2013 Christophe-Marie Duquesne <chmd@chmd.fr>
   *
   * CSS for making a resume with pandoc. Inspired by moderncv.
   *
   * This CSS document is delivered to you under the CC BY-SA 3.0 License.
   * https://creativecommons.org/licenses/by-sa/3.0/deed.en_US
   */
  
  /* Whole document */
  body {
      font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
      width: 800px;
      margin: auto;
      background: #FFFFFF;
      padding: 10px 10px 10px 10px;
  }
  
  /* Title of the resume */
  h1 {
      font-size: 55px;
      color: #757575;
      text-align:center;
      margin-bottom:15px;
  }
  h1:hover {
      background-color: #757575;
      color: #FFFFFF;
      text-shadow: 1px 1px 1px #333;
  }
  
  /* Titles of categories */
  h2 {
      /* This is called "sectioncolor" in the ConTeXt stylesheet. */
      color: #397249;
  }
  /* There is a bar just before each category */
  h2:before {
      content: "";
      display: inline-block;
      margin-right:1%;
      width: 16%;
      height: 10px;
      /* This is called "rulecolor" in the ConTeXt stylesheet. */
      background-color: #9CB770;
  }
  h2:hover {
      background-color: #397249;
      color: #FFFFFF;
      text-shadow: 1px 1px 1px #333;
  }
  
  /* Definitions */
  dl {
      width: 100%;
      overflow: hidden;hidden;
      padding: 0;
      margin: 0;
  }
  
  dt {
      float: left;
      clear: left;
      width: 17%;
      /* adjust the width; make sure the total of both is 100% */
      padding: 0;
      margin: 0
  }
  dd {
      float: left;
      width: 83%;
      /* adjust the width; make sure the total of both is 100% */
      padding: 0;
      margin: 0
  }
  p {
      margin-top:0;
      margin-bottom:7px;
  }
  
  /* Blockquotes */
  blockquote {
      text-align: center
  }
  
  /* Links */
  a {
      text-decoration: none;
      color: #397249;
  }
  a:hover, a:active {
      background-color: #397249;
      color: #FFFFFF;
      text-decoration: none;
      text-shadow: 1px 1px 1px #333;
  }
  
  /* Horizontal separators */
  hr {
      color: #A6A6A6;
  }
  
  table {
      width: 100%;
  }
  </style>
</head>
<body>
<h1 id="guangchuang-yu">Guangchuang Yu</h1>
<hr />
<table>
<tbody>
<tr class="odd">
<td align="left">School of Public Health, The University of Hong Kong</td>
<td align="right">(852)-62269693</td>
</tr>
<tr class="even">
<td align="left">L5-14, 5/F, Laboratory Block, Faculty of Medicine Building</td>
<td align="right"><a href="mailto:guangchuangyu@gmail.com">guangchuangyu@gmail.com</a></td>
</tr>
<tr class="odd">
<td align="left">22 Sassoon Road, Pokfulam, Hong Kong</td>
<td align="right"><a href="https://guangchuangyu.github.io" class="uri">https://guangchuangyu.github.io</a></td>
</tr>
</tbody>
</table>
<hr />
<p><link rel="stylesheet" href="https://guangchuangyu.github.io/css/font-awesome.min.css"></p>
<h2 id="education"><i class="fa fa-graduation-cap"></i> Education</h2>
<dl>
<dt>2013-2017</dt>
<dd><strong>PhD</strong>; The University of Hong Kong; <em>Evolution Study of Influenza A Virus</em>
</dd>
<dt>2006-2009</dt>
<dd><strong>MSc</strong>; Anhui Medical University; <em>Biochemistry and Molecular Biology</em>
</dd>
<dt>2001-2005</dt>
<dd><strong>BSc</strong>; South China Agricultural University; <em>Biotechnology</em>
</dd>
<dt>2015</dt>
<dd>Institut Pasteur, Paris; <em>Train the Trainers 2015</em>
</dd>
</dl>
<h2 id="certificate"><i class="fa fa-book"></i> CERTIFICATE</h2>
<dl>
<dt>2015</dt>
<dd><em>Certificate of Teaching and Learning in Higher Education</em>, Centre for Enhancement of Teaching and Learning, The University of Hong Kong
</dd>
</dl>
<h2 id="employment"><i class="fa fa-university"></i> EMPLOYMENT</h2>
<dl>
<dt>2017</dt>
<dd>Post-doctoral Fellow, <em>State Key Laboratory of Infectious Diseases, School of Public Health, The University of Hong Kong:</em>
</dd>
<dt>2009-2013</dt>
<dd>Bioinformatician, <em>College of Life Science and Technology, Jinan University:</em>
</dd>
</dl>
<h2 id="teaching"><i class="fa fa-slideshare"></i> Teaching</h2>
<dl>
<dt>2013-2014</dt>
<dd><strong>Student Helper</strong>; BIOL6001 Presentation Skills and Research Seminars in Biological Sciences (postgraduate course), The University of Hong Kong
</dd>
<dt>2011-2012</dt>
<dd><strong>Lecturer</strong>; Functional Proteomics (postgraduate course), Jinan University
</dd>
</dl>
<h2 id="awards"><i class="fa fa-trophy"></i> Awards</h2>
<dl>
<dt>2013-2017</dt>
<dd>Postgraduate Studentships (for PhD students at The University of Hong Kong)
</dd>
<dt>2012</dt>
<dd>Best paper of the year 2010, Journal of Integrated OMICS.
</dd>
</dl>
<h2 id="grants"><i class="fa fa-database"></i> Grants</h2>
<dl>
<dt>2013</dt>
<dd>(completed) PI, Investigate functions of hnRNPK in regulating cytokinesis of human lung cancer cell, Fundamental Research Funds for the Central Universities (21613414), ￥100,000
</dd>
<dt>2011</dt>
<dd>(completed) PI, Dynamic proteomic analysis of HeLa cell cycle, Fundamental Research Funds for the Central Universities (21611303), ￥20,000
</dd>
</dl>
<h2 id="publications"><i class="fa fa-file-text"></i> Publications</h2>
<blockquote>
<p>Citation = 1412, H-index = 14, I10-index = 18</p>
</blockquote>
<blockquote>
<p>(data from <a href="https://scholar.google.com/citations?user=DO5oG40AAAAJ&amp;hl=en">Google Scholar</a>, Mar 2018)</p>
</blockquote>
<link rel='stylesheet' href=https://guangchuangyu.github.io/resume/css/morris.css>
<script src='https://guangchuangyu.github.io/resume/css/jquery.min.js' type='text/javascript'></script>
<script src='https://guangchuangyu.github.io/resume/css/raphael-min.js' type='text/javascript'></script>
<script src='https://guangchuangyu.github.io/resume/css/morris-0.4.2.min.js' type='text/javascript'></script>
<style>
  .rChart {
    display: block;
    margin-left: auto; 
    margin-right: auto;
    width: 800px;
    height: 300px;
  }  
  </style>
<div id="chart43e43547535" class="rChart morris">

</div>
<script type='text/javascript'>
    var chartParams = {
 "element": "chart43e43547535",
"width":            800,
"height":            400,
"xkey": "year",
"ykeys": [
 "cites" 
],
"data": [
 {
 "year":           2010,
"cites":              9 
},
{
 "year":           2011,
"cites":             18 
},
{
 "year":           2012,
"cites":             62 
},
{
 "year":           2013,
"cites":             86 
},
{
 "year":           2014,
"cites":            109 
},
{
 "year":           2015,
"cites":            144 
},
{
 "year":           2016,
"cites":            269 
},
{
 "year":           2017,
"cites":            512 
},
{
 "year":           2018,
"cites":            173 
} 
],
"id": "chart43e43547535",
"labels": "cites" 
},
      chartType = "Bar"
    new Morris[chartType](chartParams)
</script>
<p>         <font color="red"><i class="fa fa-fire"></i></font> ESI Hot Paper (X1)</p>
<p>         <font color="orange"><i class="fa fa-trophy"></i></font> ESI Highly Cited Paper (X4)</p>
<p>         <em>Methods in Ecology and Evolution</em> (X1, rank 11 in <em>Environment Ecology</em>, IF = 5.708)</p>
<p>         <em>Bioinformatics</em> (X3, rank 4 in <em>Computer Science</em>, IF = 7.307)</p>
<p>         (<sup>†</sup> indicates equal contribution, <sup>*</sup> indicates corresponding)</p>
<h3 id="journal-articles"><strong>Journal Articles</strong></h3>
<ol style="list-style-type: decimal">
<li><strong>G Yu</strong>. <a href="https://www.biorxiv.org/content/early/2018/01/31/256784">clusterProfiler: universal enrichment tool for functional and comparative study</a>. <strong><em>bioRxiv</em></strong>. 2018, <a href="https://doi.org/10.1101/256784" class="uri">https://doi.org/10.1101/256784</a></li>
<li><strong>G Yu</strong>, DK Smith, H Zhu, Y Guan, TTY Lam<sup>*</sup>. <a href="http://onlinelibrary.wiley.com/doi/10.1111/2041-210X.12628/abstract">ggtree: an R package for visualization and annotation of phylogenetic trees with their covariates and other associated data</a>. <strong><em>Methods in Ecology and Evolution</em></strong>. 2017, 8(1):28-36. <font color="red"><i class="fa fa-fire"></i></font> <font color="orange"><i class="fa fa-trophy"></i></font> <a href="https://scholar.google.com.hk/scholar?oi=bibs&amp;hl=en&amp;cites=14021087557216297475"><img src="https://img.shields.io/badge/cited%20by-95-blue.svg?style=flat" alt="citation" /></a></li>
<li><strong>G Yu</strong>, QY He<sup>*</sup>. <a href="http://pubs.rsc.org/en/Content/ArticleLanding/2016/MB/C5MB00663E">ReactomePA: an R/Bioconductor package for reactome pathway analysis and visualization</a>. <strong><em>Molecular BioSystems</em></strong>. 2016, 12(2):477-479. <font color="orange"><i class="fa fa-trophy"></i></font> <a href="https://scholar.google.com.hk/scholar?oi=bibs&amp;hl=en&amp;cites=3311691878690959578"><img src="https://img.shields.io/badge/cited%20by-63-blue.svg?style=flat" alt="citation" /></a></li>
<li><strong>G Yu</strong><sup>*</sup>, LG Wang, QY He<sup>*</sup>. <a href="http://bioinformatics.oxfordjournals.org/cgi/content/abstract/btv145">ChIPseeker: an R/Bioconductor package for ChIP peak annotation, comparision and visualization</a>. <strong><em>Bioinformatics</em></strong>. 2015, 31(14):2382-2383. <font color="orange"><i class="fa fa-trophy"></i></font> <a href="https://scholar.google.com.hk/scholar?oi=bibs&amp;hl=en&amp;cites=12053363057899219488"><img src="https://img.shields.io/badge/cited%20by-108-blue.svg?style=flat" alt="citation" /></a></li>
<li><strong>G Yu</strong><sup>*</sup>, LG Wang, GR Yan, QY He<sup>*</sup>. <a href="http://bioinformatics.oxfordjournals.org/cgi/content/abstract/btu684">DOSE: an R/Bioconductor package for Disease Ontology Semantic and Enrichment analysis</a>. <strong><em>Bioinformatics</em></strong>. 2015, 31(4):608-609. <a href="https://scholar.google.com.hk/scholar?oi=bibs&amp;hl=en&amp;cites=10904694463327099987"><img src="https://img.shields.io/badge/cited%20by-65-blue.svg?style=flat" alt="citation" /></a></li>
<li><strong>G Yu</strong><sup>†</sup>, LG Wang<sup>†</sup>, XH Meng, QY He<sup>*</sup>. <a href="http://www.biomedcentral.com/1756-0500/5/360/">LXtoo: an integrated live Linux distribution for the bioinformatics community</a>. <strong><em>BMC Research Notes</em></strong>. 2012, 5(1):360.</li>
<li><strong>G Yu</strong>, LG Wang, Y Han, QY He<sup>*</sup>. <a href="http://online.liebertpub.com/doi/abs/10.1089/omi.2011.0118">clusterProfiler: an R package for comparing biological themes among gene clusters</a>. <strong><em>OMICS: A Journal of Integrative Biology</em></strong>. 2012, 16(5):284-287. <font color="orange"><i class="fa fa-trophy"></i></font> <a href="https://scholar.google.com.hk/scholar?oi=bibs&amp;hl=en&amp;cites=2349076811020942117"><img src="https://img.shields.io/badge/cited%20by-439-blue.svg?style=flat" alt="citation" /></a></li>
<li><strong>G Yu</strong>, QY He<sup>*</sup>. <a href="http://www.jclinbioinformatics.com/content/1/1/15/abstract">Functional similarity analysis of human virus-encoded miRNAs</a>. <strong><em>Journal of Clinical Bioinformatics</em></strong>. 2011, 1(1):15.</li>
<li><strong>G Yu</strong><sup>†</sup>, CL Xiao<sup>†</sup>, X Bo, CH Lu, Y Qin, S Zhan, QY He<sup>*</sup>. <a href="http://www.jiomics.com/index.php/jio/article/view/21">A new method for measuring functional similarity of microRNAs</a>. <strong><em>Journal of Integrated OMICS</em></strong>. 2011, 1(1):49-54.</li>
<li><strong>G Yu</strong><sup>†</sup>, CL Xiao<sup>†</sup>, CH Lu<sup>†</sup>, HT Jia, F Ge, W Wang, XF Yin, HL Jia, JX He<sup>*</sup>, QY He<sup>*</sup>. <a href="http://pubs.rsc.org/en/Content/ArticleLanding/2011/MB/C0MB00055H">Phosphoproteome profile of human lung cancer cell line A549</a>. <strong><em>Molecular BioSystems</em></strong>. 2011, 7(2):472-479.</li>
<li><strong>G Yu</strong><sup>†</sup>, F Li<sup>†</sup>, Y Qin, X Bo<sup>*</sup>, Y Wu, S Wang<sup>*</sup>. <a href="http://bioinformatics.oxfordjournals.org/cgi/content/abstract/26/7/976">GOSemSim: an R package for measuring semantic similarity among GO terms and gene products</a>. <strong><em>Bioinformatics</em></strong>. 2010, 26(7):976-978. <a href="https://scholar.google.com.hk/scholar?oi=bibs&amp;hl=en&amp;cites=9484177541993722322"><img src="https://img.shields.io/badge/cited%20by-326-blue.svg?style=flat" alt="citation" /></a></li>
<li>X Sun<sup>†</sup>, <strong>G Yu</strong><sup>†</sup>, Q Xu, N Li, C Xiao, X Yin, K Cao, J Han, QY He<sup>*</sup>. <a href="http://pubs.rsc.org/en/content/articlelanding/2013/mt/c3mt00126a">Putative cobalt- and nickel-binding proteins and motifs in <em>Streptococcus pneumonia</em></a>. <strong><em>Metallomics</em></strong>. 2013, 5(7):928-935.</li>
<li>YY Zhu<sup>†</sup>, <strong>G Yu</strong><sup>†</sup>, Y Zhang, Z Xu, YQ Huang, GR Yan<sup>*</sup>, QY He<sup>*</sup>. <a href="http://onlinelibrary.wiley.com/doi/10.1002/pmic.201200273/abstract">A novel andrographolide derivative AL-1 exerts its cytotoxicity on K562 cells through a ROS-dependent mechanism</a>. <strong><em>Proteomics</em></strong>, 2013, 13(1):169-178. <!--
    987. M Tang, S Weng, __G Yu__, K Yang, M Pan, S He, Q Liu^\*^. [RE: Sequence features of the virulence protein VP24 of Ebola virus](http://science.sciencemag.org/content/351/6279/1339/tab-e-letters). __*Science*__. 2017.
    --></li>
<li>XY Yang, K He, G Du, X Wu, <strong>G Yu</strong>, Y Pan, G Zhang<sup>*</sup>, X Sun<sup>*</sup>, QY He<sup>*</sup>. <a href="http://journal.frontiersin.org/article/10.3389/fmicb.2016.00078/full">Integrated Translatomics with Proteomics to Identify Novel Iron–Transporting Proteins in <em>Streptococcus pneumoniae</em></a>. <strong><em>Frontiers in Microbiology</em></strong>. 2016, 7:78.</li>
<li>PY Huang, CCD Lee, CH Yip, CL Cheung, <strong>G Yu</strong>, TTY Lam, DK Smith, H Zhu<sup>*</sup>, Y Guan<sup>*</sup>. <a href="http://www.sciencedirect.com/science/article/pii/S1567134815300721">Genetic characterization of highly pathogenic H5 influenza viruses from poultry in Taiwan, 2015</a>. <strong><em>Infection, Genetics and Evolution</em></strong>. 2016, 38:96-100.</li>
<li>SH Xu, JZ Huang, ML Xu, <strong>G Yu</strong>, XF Yin, D Chen<sup>*</sup>, GR Yan<sup>*</sup>. <a href="http://onlinelibrary.wiley.com/doi/10.1002/path.4515/abstract">ACK1 Promotes Gastric Cancer Epithelial-Mesenchymal Transition and Metastasis through AKT-POU2F1-ECD Signaling</a>. <strong><em>The Journal of Pathology</em></strong>. 2015, 236(2):175-185.</li>
<li>Z Guo, J Han, XY Yang, K Cao, K He, G Du, G Zeng, L Zhang, <strong>G Yu</strong>, Z Sun, QY He<sup>*</sup>, X Sun<sup>*</sup>. <a href="http://pubs.rsc.org/en/content/articlelanding/2015/mt/C4MT00276H">Proteomic analysis on the copper resistance of <em>Streptococcus pneumonia</em></a>. <strong><em>Metallomics</em></strong>. 2015, 7(3):448-454.</li>
<li>XY Yang, L Zhang, J Liu, N Li, <strong>G Yu</strong>, K Cao, J Han, G Zeng, Y Pan, X Sun<sup>*</sup>, QY He<sup>*</sup>. <a href="http://www.sciencedirect.com/science/article/pii/S1874391914005521">Proteomic analysis on the antibacterial activity of a Ru(II) complex against <em>Strepococcus pneumonia</em></a>. <strong><em>Journal of Proteomics</em></strong>. 2015, 115:107-116.</li>
<li>N Wang<sup>†</sup>, J Zhang<sup>†</sup>, L Zhang, XY Yang, N Li, <strong>G Yu</strong>, J Han, K Cao, Z Guo, X Sun<sup>*</sup>, QY He<sup>*</sup>. <a href="http://pubs.rsc.org/en/content/articlelanding/2014/mt/c4mt00027g">Proteomic analysis of putative heme-binding proteins in <em>Streptococcus pyogenes</em></a>. <strong><em>Metallomics</em></strong>. 2014, 6(8):1451-1459.</li>
<li>H Sun<sup>*</sup>, J Weng, <strong>G Yu</strong>, RH Massawe. <a href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0077090">A DNA-Based Semantic Fusion Model for Remote Sensing Data</a>. <strong><em>PLoS One</em></strong>. 2013, 8(10):e77090.</li>
<li>GR Yan<sup>†</sup>, Z Tan<sup>†</sup>, Y Wang, ML Xu, <strong>G Yu</strong>, Y Li<sup>*</sup>, QY He<sup>*</sup>. <a href="http://onlinelibrary.wiley.com/doi/10.1002/pmic.201300152/abstract">Quantitative Proteomics Characterization on the Anti-tumor Effects of Isodexyelephantopin against Nasopharyngeal Carcinoma</a>. <strong><em>Proteomics</em></strong>. 2013, 13(21):3222-3232.</li>
<li>J Li, X Lu, Q Wu, <strong>G Yu</strong>, Z Xu, L Qiu, Z Pei, Y Lin, J Pang<sup>*</sup>. <a href="http://www.eurekaselect.com/107734/article">Design, SAR, angiogenic activities evaluation and pro-angiogenic mechanism of new marine cyclopeptide analogs</a>. <strong><em>Current Medicinal Chemistry</em></strong>. 2013, 20(9):1183-1194.</li>
<li>Y Han, <strong>G Yu</strong>, H Sarioglu, A Caballero-Martinez, F Schlott, M Ueffing, H Haase, C Peschel, AM Krackhardt<sup>*</sup>. <a href="http://www.sciencedirect.com/science/article/pii/S1874391912007567">Proteomic investigation of the interactome of FMNL1 in hematopoietic cells unveils a role in calcium-dependent membrane plasticity</a>. <strong><em>Journal of Proteomics</em></strong>. 2013, 78:72-82.</li>
<li>Y Ma<sup>†</sup>, J Yang<sup>†</sup>, X Fan<sup>†</sup>, H Zhao, W Hu, Z Li, <strong>G Yu</strong>, X Ding, J Wang, X Bo, X Zheng, Z Zhou<sup>*</sup>, S Wang<sup>*</sup>. <a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1582-4934.2012.01572.x/abstract">Cellular MicroRNA let-7c Inhibits M1 Protein Expression of the H1N1 Influenza A Virus in Infected Human Lung Epithelial Cells</a>. <strong><em>Journal of Cellular and Molecular Medicine</em></strong>. 2012, 16(10):2539-2546.</li>
<li>GR Yan<sup>†</sup>, FY Zhou<sup>†</sup>, BL Dang, Y Zhang, <strong>G Yu</strong>, X Liu, QY He<sup>*</sup>. <a href="http://onlinelibrary.wiley.com/doi/10.1002/pmic.201100652/abstract">Genistein-induced mitotic arrest of gastric cancer cells by downregulating KIF20A, a proteomics study</a>. <strong><em>Proteomics</em></strong>. 2012, 12(14):2391-2399.</li>
<li>X Sun<sup>†</sup>, XY Yang<sup>†</sup>, XF Yin, <strong>G Yu</strong>, CL Xiao, X He, QY He<sup>*</sup>. <a href="http://www.liebertonline.com/doi/abs/10.1089/omi.2010.0133">Proteomic Analysis of Membrane Proteins from <em>Streptococcus pneumoniae</em> with Multiple Separation Methods plus High Accuracy Mass Spectrometry</a>. <strong><em>OMICS: A Journal of Integrative Biology</em></strong>. 2011, 15(10):683-694.</li>
</ol>
<h3 id="books"><strong>Books</strong></h3>
<ol style="list-style-type: decimal">
<li><strong>G Yu</strong>, &quot;Bioinformatics II&quot;, in &quot;Functional Protein Research&quot;, 385-425, Qing-Yu He (<em>Eds</em>), <strong><em>Science Press</em></strong>, 2012. (China’s Twelfth Five-Year National Important Books Project, in Chinese)</li>
</ol>
<h2 id="conference-proceedings"><i class="fa fa-group"></i> Conference Proceedings</h2>
<h3 id="presentations"><strong>Presentations</strong></h3>
<ol style="list-style-type: decimal">
<li><strong>ggtree for tree visualization &amp; annotation</strong> (invited), Phylogenetic tree visualization workshop/hackathon, The Field Museum of Natural History, Chicago, May. 2017</li>
<li><strong>Computational Methods in Ecology and Evolution</strong>, The Second International Workshop on Environmental Microbiomes, Sun Yat-sen University, Guangzhou, April. 2017</li>
<li><strong><a href="https://guangchuangyu.github.io/presentation/2016-ggtree-chinar/">ggtree for visualization and annotation of phylogenetic trees</a></strong> (invited), The 9<sup>th</sup> China-R Conference, Renmin University of China, Beijing, May. 2016</li>
</ol>
<h3 id="posters"><strong>Posters</strong></h3>
<ol style="list-style-type: decimal">
<li><strong>ggtree: a programming package for complex data integration and analysis in phylogeny to identify potential genetic markers of influenza a virus that determine interspecies transmission</strong>, Scientific Advisory Board (SAB) Meeting for the Theme-based Research Scheme on Viral, Host and Environmental Determinants of Influenza Virus Transmission and Pathogenesis, The University of Hong Kong, Hong Kong, Jun. 2017</li>
<li><strong>Phylogeny-guided genome assembly method for short read nucleotide sequences from co-infecting influenza A viruses</strong>, Options IX for the control of influenza, International Society for Influenza and other Respiratory Virus Disease, Chicago, Aug. 2016</li>
<li><strong>ggtree: an <em>R</em> package for visualization and annotation of phylogenetic tree with different types of meta-data</strong>, The 20<sup>th</sup> Research Postgraduate Symposium, The University of Hong Kong, Hong Kong, Dec. 2015</li>
</ol>
<h2 id="public-media"><i class="fa fa-video-camera"></i> Public Media</h2>
<p>&quot;Plotting trees + data&quot; lecture by Guangchuang Yu. <a href="https://youtu.be/tqspAXM0UQE" class="uri">https://youtu.be/tqspAXM0UQE</a></p>
<h2 id="professional-affiliations"><i class="fa fa-home"></i> Professional Affiliations</h2>
<dl>
<dt>2016</dt>
<dd><strong>Member</strong>; International Society for Influenza and other Respiratory Virus Diseases
</dd>
<dt>2015</dt>
<dd><strong>Steering Committee</strong>; International Network for Data Analysis, Institut Pasteur
</dd>
<dt>2005-2009</dt>
<dd><strong>Visiting Student</strong>; Beijing Institute of Radiation Medicine
</dd>
</dl>
<h2 id="professional-services"><i class="fa fa-user"></i> Professional Services</h2>
<dl>
<dt><strong>Associate Editor</strong></dt>
<dd>Journal of Integrated OMICS, 2011-
</dd>
<dt><strong>Reviewer</strong></dt>
<dd><p>Bioinformatics<br></p>
<p>BMC Bioinformatics<br></p>
<p>Computational Biology and Chemistry<br></p>
<p>Current Bioinformatics<br></p>
<p>Gastroenterology Research and Practice<br></p>
<p>Molecular BioSystems<br></p>
<p>R Journal</p>
</dd>
</dl>
<h2 id="software-development"><i class="fa fa-gears"></i> Software Development</h2>
<blockquote>
<p><strong>R ranking</strong>: ranked 47/71749 worldwide by <a href="http://git-awards.com/users/guangchuangyu">Git Awards</a></p>
</blockquote>
<dl>
<dt><a href="https://cran.r-project.org/package=badger">badger</a></dt>
<dd>Badge for R Package
</dd>
<dt><a href="https://cran.r-project.org/package=dlstats">dlstats</a></dt>
<dd>Download Stats of R Packages
</dd>
<dt><a href="https://guangchuangyu.github.io/emojifont">emojifont</a></dt>
<dd>Emoji and Font Awesome in Graphics
</dd>
<dt><a href="https://cran.r-project.org/package=ggimage">ggimage</a></dt>
<dd>Use Image in 'ggplot2'
</dd>
<dt><a href="http://cran.r-project.org/package=meme">meme</a></dt>
<dd>Create Meme
</dd>
<dt><a href="https://cran.r-project.org/package=rvcheck">rvcheck</a></dt>
<dd>R/Package Version Check
</dd>
<dt><a href="https://cran.r-project.org/package=scatterpie">scatterpie</a></dt>
<dd>Scatter Pie Plot
</dd>
<dt><a href="http://cran.r-project.org/package=shadowtext">shadowtext</a></dt>
<dd>Shadow Text Grob and Layer
</dd>
<dt><a href="https://guangchuangyu.github.io/ChIPseeker">ChIPseeker</a></dt>
<dd>ChIPseeker for ChIP peak Annotation, Comparison, and Visualization
</dd>
<dt><a href="https://guangchuangyu.github.io/clusterProfiler">clusterProfiler</a></dt>
<dd>statistical analysis and visualization of functional profiles for genes and gene clusters
</dd>
<dt><a href="https://guangchuangyu.github.io/DOSE">DOSE</a></dt>
<dd>Disease Ontology Semantic and Enrichment analysis
</dd>
<dt><a href="https://guangchuangyu.github.io/ggtree">ggtree</a></dt>
<dd>an R package for visualization and annotation of phylogenetic trees with their covariates and other associated data
</dd>
<dt><a href="https://guangchuangyu.github.io/GOSemSim">GOSemSim</a></dt>
<dd>GO-terms Semantic Similarity Measures
</dd>
<dt><a href="https://guangchuangyu.github.io/meshes">meshes</a></dt>
<dd>MeSH Enrichment and Semantic analyses
</dd>
<dt><a href="https://guangchuangyu.github.io/ReactomePA">ReactomePA</a></dt>
<dd>Reactome Pathway Analysis
</dd>
<dt><a href="https://www.bioconductor.org/packages/seqcombo">seqcombo</a></dt>
<dd>Visualization Tool for Sequence Recombination and Reassortment
</dd>
<dt><a href="https://cran.r-project.org/package=tidytree">tidytree</a></dt>
<dd>A Tidy Tool for Phylogenetic Tree Data Manipulation
</dd>
<dt><a href="https://github.com/id-bioinfo/TIPars">TIPars</a></dt>
<dd>Taxa Insertion by Parsimony
</dd>
<dt><a href="https://bioconductor.org/packages/treeio">treeio</a></dt>
<dd>Base Classes and Functions for Phylogenetic Tree Input and Output
</dd>
</dl>
<h3 id="contributions-to-open-source-libraries"><strong>Contributions to open source libraries</strong></h3>
<dl>
<dt><a href="https://cran.r-project.org/package=basicTrendline">basicTrendline</a></dt>
<dd>Add Trendline of Basic Regression Models to Plot
</dd>
<dt><a href="https://cran.r-project.org/package=cowsay">cowsay</a></dt>
<dd>Messages, Warnings, Strings with Ascii Animals
</dd>
<dt><a href="https://cran.r-project.org/package=kableExtra">kableExtra</a></dt>
<dd>Construct Complex Table with 'kable' and Pipe Syntax
</dd>
<dt><a href="https://github.com/lch14forever/microbiomeViz">microbiomeViz</a></dt>
<dd>Visualizing Microbiome Data
</dd>
<dt><a href="https://bioconductor.org/packages/MSnbase/">MSnbase</a></dt>
<dd>Base Functions and Classes for Mass Spectrometry and Proteomics
</dd>
<dt><a href="https://github.com/jkeirstead/scholar">scholar</a></dt>
<dd>Analyse Citation Data from Google Scholar
</dd>
</dl>
</body>
</html>
