---
output:
  pdf_document:
    latex_engine: pdflatex
    template: rap-latex-cv.tex
geometry: margin=1in

title: "CV"
author: Guangchuang Yu
orcid: "orcid.org/0000-0002-6485-8781"
jobtitle: "Post-doctoral Fellow"
address: "L514, Laboratory Block, Faculty of Medicine Building, 22 Sassoon Road, Pokfulam, Hong Kong"
fontawesome: yes
email: guangchuangyu@gmail.com
github: GuangchuangYu
phone: "+852 6226 9693"
web: 'guangchuangyu.github.io'
updated: no

keywords: RMarkdown, academic CV, template

fontfamily: mathpazo
fontfamilyoptions: sc, osf
fontsize: 10pt
linkcolor: blue
urlcolor: blue
---

# \faGraduationCap \hspace{2pt} EDUCATION

*The University of Hong Kong* \hspace{40pt} Ph.D. Evolution Study of Influenza A virus \hfill 2017

*Anhui Medical University* \hspace{55pt} M.S. Biochemistry and Molecular Biology \hfill 2009

*South China Agricultural University* \hspace{10pt} B.S. Biotechnology \hfill 2005

*Institut Pasteur, Paris* \hspace{70pt} Train the Trainers \hfill 2015

# \faBook \hspace{2pt} CERTIFICATE

*Certificate of Teaching and Learning in Higher Education*, Centre for Enhancement of Teaching and Learning, The University of Hong Kong \hfill 2015

# \faUniversity \hspace{2pt} EMPLOYMENT

*State Key Laboratory of Infectious Diseases, School of Public Health, The University of Hong Kong:*

 > Post-doctoral Fellow \hfill 2017

*College of Life Science and Technology, Jinan University:*

 > Bioinformatician \hfill 2009--2013

# \faSlideshare \hspace{2pt} TEACHING

*Student Helper, The University of Hong Kong* \hfill 2013--2014

> BIOL6001 Presentation Skills and Research Seminars in Biological Sciences

*Lecturer, Jinan University* \hfill 2011--2012

> Functional Proteomics

# \faTrophy \hspace{2pt} AWARDS

**2013--2017**  *Postgraduate Studentships*. The University of Hong Kong.

**2012**  *Best paper of the year 2010*. Journal of Integrated OMICS.

# \faDatabase \hspace{2pt} GRANTS

**2013**  *(completed) PI*, Investigate functions of hnRNPK in regulating cytokinesis of human lung cancer cell. Fundamental Research Funds for the Central Universities (21613414), CNY 100,000

**2011**  *(completed) PI*, Dynamic proteomic analysis of HeLA cell cycle, Fundamental Research Funds for the Central Universities (21611303), CNY 20,000

# \faFileText \hspace{2pt} PUBLICATIONS

## **Journal Articles**

```{r echo=F, message=F, fig.asp=0.618}
library(methods)
library(scholar)
library(ggplot2)
library(ggthemes)
library(ypages)

cite <- function(article_id) {
    sprintf("cited by **%d**", sum(get_article_cite_history("DO5oG40AAAAJ", article_id)$cites))
}

id <- 'DO5oG40AAAAJ'
ygc <- get_profile(id)
df <- get_citation_history(id)

dd <- strsplit(date(), "\\s+")[[1]]
subtitle <- paste("data from Google Scholar,", dd[2], dd[5])
title <- paste0("Citation = ", ygc$total_cites, ", H-index = ", ygc$h_index, ", I10-index = ", ygc$i10_index)
ggplot(df, aes(year, cites)) +
  geom_segment(aes(xend=year, yend=0), size=1, color='grey') +
  geom_point(size=8, color="#96B56C") +
  theme_minimal() +
  geom_text(aes(label=cites), size=3) +
  xlab(NULL) + ylab(NULL) +
	ylim(NA, round(max(df$cites) * 1.1)) +
  ggtitle(title) +
	scale_x_continuous(breaks = df$year) +
  theme(plot.title=element_text(size=10)) +
  labs(caption=subtitle) +
  theme(panel.grid.major.x = element_blank(),
        panel.grid.minor = element_blank())
```


> \emph{\color{OrangeRed}\faFire} ESI Hot Paper (X2)

> \emph{\color{BurntOrange}\faTrophy} ESI Highly Cited Paper (X5)

> *Methods in Ecology and Evolution* (X1, rank 11 in *Environment Ecology*, IF = 5.708)

> *Bioinformatics* (X3, rank 4 in *Computer Science*, IF = 7.307)

> (^†^ indicates equal contribution, ^\*^ indicates corresponding)


1. __G Yu__, DK Smith, H Zhu, Y Guan, TTY Lam^\*^. [ggtree: an R package for visualization and annotation of phylogenetic trees with their covariates and other associated data](http://onlinelibrary.wiley.com/doi/10.1111/2041-210X.12628/abstract). __*Methods in Ecology and Evolution*__. 2017, 8(1):28-36. \emph{\color{OrangeRed}\faFire} \emph{\color{BurntOrange}\faTrophy}
 \hfill `r cite("HtEfBTGE9r8C")`
92. __G Yu__, QY He^\*^. [ReactomePA: an R/Bioconductor package for reactome pathway analysis and visualization](http://pubs.rsc.org/en/Content/ArticleLanding/2016/MB/C5MB00663E). __*Molecular BioSystems*__. 2016, 12(2):477-479. \emph{\color{BurntOrange}\faTrophy}
 \hfill `r cite("8d8msizDQcsC")`
93. __G Yu__^\*^, LG Wang, QY He^\*^. [ChIPseeker: an R/Bioconductor package for ChIP peak annotation, comparision and visualization](http://bioinformatics.oxfordjournals.org/cgi/content/abstract/btv145). __*Bioinformatics*__. 2015, 31(14):2382-2383. \emph{\color{OrangeRed}\faFire} \emph{\color{BurntOrange}\faTrophy}
 \hfill `r cite("9pM33mqn1YgC")`
94.	__G Yu__^\*^, LG Wang, GR Yan, QY He^\*^. [DOSE: an R/Bioconductor package for Disease Ontology Semantic and Enrichment analysis](http://bioinformatics.oxfordjournals.org/cgi/content/abstract/btu684). __*Bioinformatics*__. 2015, 31(4):608-609.
 \hfill `r cite("Ug5p-4gJ2f0C")`
95.	__G Yu__^†^, LG Wang^†^, XH Meng, QY He^\*^. [LXtoo: an integrated live Linux distribution for the bioinformatics community](http://www.biomedcentral.com/1756-0500/5/360/). __*BMC Research Notes*__. 2012, 5(1):360.
96.	__G Yu__, LG Wang, Y Han, QY He^\*^. [clusterProfiler: an R package for comparing biological themes among gene clusters](http://online.liebertpub.com/doi/abs/10.1089/omi.2011.0118). __*OMICS: A Journal of Integrative Biology*__. 2012, 16(5):284-287. \emph{\color{BurntOrange}\faTrophy}
 \hfill `r cite("MLfJN-KU85MC")`
97.	__G Yu__, QY He^\*^. [Functional similarity analysis of human virus-encoded miRNAs](http://www.jclinbioinformatics.com/content/1/1/15/abstract). __*Journal of Clinical Bioinformatics*__. 2011, 1(1):15.
98.	__G Yu__^†^, CL Xiao^†^, X Bo, CH Lu, Y Qin, S Zhan, QY He^\*^. [A new method for measuring functional similarity of microRNAs](http://www.jiomics.com/index.php/jio/article/view/21). __*Journal of Integrated OMICS*__. 2011, 1(1):49-54.
99.	__G Yu__^†^, CL Xiao^†^, CH Lu^†^, HT Jia, F Ge, W Wang, XF Yin, HL Jia, JX He^\*^, QY He^\*^. [Phosphoproteome profile of human lung cancer cell line A549](http://pubs.rsc.org/en/Content/ArticleLanding/2011/MB/C0MB00055H). __*Molecular BioSystems*__. 2011, 7(2):472-479.
100. __G Yu__^†^, F Li^†^, Y Qin, X Bo^\*^, Y Wu, S Wang^\*^. [GOSemSim: an R package for measuring semantic similarity among GO terms and gene products](http://bioinformatics.oxfordjournals.org/cgi/content/abstract/26/7/976). __*Bioinformatics*__. 2010, 26(7):976-978. \emph{\color{BurntOrange}\faTrophy}
 \hfill `r cite("tuHXwOkdijsC")`
199. X Sun^†^, __G Yu__^†^, Q Xu, N Li, C Xiao, X Yin, K Cao, J Han, QY He^\*^. [Putative cobalt- and nickel-binding proteins and motifs in *Streptococcus pneumonia*](http://pubs.rsc.org/en/content/articlelanding/2013/mt/c3mt00126a). __*Metallomics*__. 2013, 5(7):928-935.
200. YY Zhu^†^, __G Yu__^†^, Y Zhang, Z Xu, YQ Huang, GR Yan^\*^, QY He^\*^. [A novel andrographolide derivative AL-1 exerts its cytotoxicity on K562 cells through a ROS-dependent mechanism](http://onlinelibrary.wiley.com/doi/10.1002/pmic.201200273/abstract). __*Proteomics*__, 2013, 13(1):169-178.
988. XY Yang, K He, G Du, X Wu, __G Yu__, Y Pan, G Zhang^\*^, X Sun^\*^, QY He^\*^. [Integrated Translatomics with Proteomics to Identify Novel Iron–Transporting Proteins in *Streptococcus pneumoniae*](http://journal.frontiersin.org/article/10.3389/fmicb.2016.00078/full). __*Frontiers in Microbiology*__. 2016, 7:78.
989. PY Huang, CCD Lee, CH Yip, CL Cheung, __G Yu__, TTY Lam, DK Smith, H Zhu^\*^, Y Guan^\*^. [Genetic characterization of highly pathogenic H5 influenza viruses from poultry in Taiwan, 2015](http://www.sciencedirect.com/science/article/pii/S1567134815300721). __*Infection, Genetics and Evolution*__. 2016, 38:96-100.
990. SH Xu, JZ Huang, ML Xu, __G Yu__, XF Yin, D Chen^\*^, GR Yan^\*^. [ACK1 Promotes Gastric Cancer Epithelial-Mesenchymal Transition and Metastasis through AKT-POU2F1-ECD Signaling](http://onlinelibrary.wiley.com/doi/10.1002/path.4515/abstract). __*The Journal of Pathology*__. 2015, 236(2):175-185.
991. Z Guo, J Han, XY Yang, K Cao, K He, G Du, G Zeng, L Zhang, __G Yu__, Z Sun, QY He^\*^, X Sun^\*^. [Proteomic analysis on the copper resistance of *Streptococcus pneumonia*](http://pubs.rsc.org/en/content/articlelanding/2015/mt/C4MT00276H). __*Metallomics*__. 2015, 7(3):448-454.
992. XY Yang, L Zhang, J Liu, N Li, __G Yu__, K Cao, J Han, G Zeng, Y Pan, X Sun^\*^, QY He^\*^. [Proteomic analysis on the antibacterial activity of a Ru(II) complex against *Strepococcus pneumonia*](http://www.sciencedirect.com/science/article/pii/S1874391914005521). __*Journal of Proteomics*__. 2015, 115:107-116.
993. N Wang^†^, J Zhang^†^, L Zhang, XY Yang, N Li, __G Yu__, J Han, K Cao, Z Guo, X Sun^\*^, QY He^\*^. [Proteomic analysis of putative heme-binding proteins in *Streptococcus pyogenes*](http://pubs.rsc.org/en/content/articlelanding/2014/mt/c4mt00027g). __*Metallomics*__. 2014, 6(8):1451-1459.
994. H Sun^\*^, J Weng, __G Yu__, RH Massawe. [A DNA-Based Semantic Fusion Model for Remote Sensing Data](http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0077090). __*PLoS One*__. 2013, 8(10):e77090.
995. GR Yan^†^, Z Tan^†^, Y Wang, ML Xu, __G Yu__, Y Li^\*^, QY He^\*^. [Quantitative Proteomics Characterization on the Anti-tumor Effects of Isodexyelephantopin against Nasopharyngeal Carcinoma](http://onlinelibrary.wiley.com/doi/10.1002/pmic.201300152/abstract). __*Proteomics*__. 2013, 13(21):3222-3232.
996. J Li, X Lu, Q Wu, __G Yu__, Z Xu, L Qiu, Z Pei, Y Lin, J Pang^\*^. [Design, SAR, angiogenic activities evaluation and pro-angiogenic mechanism of new marine cyclopeptide analogs](http://www.eurekaselect.com/107734/article). __*Current Medicinal Chemistry*__. 2013, 20(9):1183-1194.
997. Y Han, __G Yu__, H Sarioglu, A Caballero-Martinez, F Schlott, M Ueffing, H Haase, C Peschel, AM Krackhardt^\*^. [Proteomic investigation of the interactome of FMNL1 in hematopoietic cells unveils a role in calcium-dependent membrane plasticity](http://www.sciencedirect.com/science/article/pii/S1874391912007567). __*Journal of Proteomics*__. 2013, 78:72-82.
998. Y Ma^†^, J Yang^†^, X Fan^†^, H Zhao, W Hu, Z Li, __G Yu__, X Ding, J Wang, X Bo, X Zheng, Z Zhou^\*^, S Wang^\*^. [Cellular MicroRNA let-7c Inhibits M1 Protein Expression of the H1N1 Influenza A Virus in Infected Human Lung Epithelial Cells](http://onlinelibrary.wiley.com/doi/10.1111/j.1582-4934.2012.01572.x/abstract). __*Journal of Cellular and Molecular Medicine*__. 2012, 16(10):2539-2546.
999. GR Yan^†^, FY Zhou^†^, BL Dang, Y Zhang, __G Yu__, X Liu, QY He^\*^. [Genistein-induced mitotic arrest of gastric cancer cells by downregulating KIF20A, a proteomics study](http://onlinelibrary.wiley.com/doi/10.1002/pmic.201100652/abstract). __*Proteomics*__. 2012, 12(14):2391-2399.
1000. X Sun^†^, XY Yang^†^, XF Yin, __G Yu__, CL Xiao, X He, QY He^\*^. [Proteomic Analysis of Membrane Proteins from *Streptococcus pneumoniae* with Multiple Separation Methods plus High Accuracy Mass Spectrometry](http://www.liebertonline.com/doi/abs/10.1089/omi.2010.0133). __*OMICS: A Journal of Integrative Biology*__. 2011, 15(10):683-694.

## **E-Letters**

1. Y Wan^†^, __G Yu__^†^, J Zou^†^, W Liu^†^, Q Liu^\*^, Why animals
   have escaped the Indian Ocean tsunami? __*Science*__. 2017,
   http://science.sciencemag.org/content/356/6340/841/tab-e-letters
100. M Tang, S Weng, __G Yu__, K Yang, M Pan, S He,
   Q Liu^\*^, RE: Sequence features of the virulence protein VP24 of Ebola
   virus. __*Science*__. 2017,
   http://science.sciencemag.org/content/351/6279/1339/tab-e-letters


## **Books**

  - __G Yu__, "Bioinformatics II", in "Functional Protein Research", 385-425, Qing-Yu He (*Eds*), __*Science Press*__, 2012. (China’s Twelfth Five-Year National Important Books Project, in Chinese)

# \faGroup \hspace{2pt} CONFERENCE PROCEEDINGS

## **Presentations**

1. **ggtree for tree visualization & annotation** (invited),
    Phylogenetic tree visualization workshop/hackathon, The Field Museum of
    Natural History, Chicago, May. 2017
100. **Computational Methods in Ecology and Evolution**,
    The Second International Workshop on Environmental Microbiomes, Sun
    Yat-sen University, Guangzhou, April. 2017
101. **[ggtree for visualization and annotation of phylogenetic trees](https://guangchuangyu.github.io/presentation/2016-ggtree-chinar/)** (invited),
	The 9^th^ China-R Conference, Renmin University of China, Beijing, May. 2016

## **Posters**

1. **ggtree: a programming package for complex data integration and analysis in
   phylogeny to identify potential genetic markers of influenza a virus that
   determine interspecies transmission**, Scientific Advisory Board (SAB) Meeting for the Theme-based Research
   Scheme on Viral, Host and Environmental Determinants of Influenza Virus
   Transmission and Pathogenesis, The University of Hong Kong, Hong Kong, Jun. 2017
100. **Phylogeny-guided genome assembly method for short read nucleotide sequences from co-infecting influenza A viruses**,
	Options IX for the control of influenza, International Society for Influenza and other Respiratory Virus Disease, Chicago, Aug. 2016
101. **ggtree: an *R* package for visualization and annotation of phylogenetic tree with different types of meta-data**,
	The 20^th^ Research Postgraduate Symposium, The University of Hong Kong, Hong Kong, Dec. 2015

# \faVideoCamera \hspace{2pt} PUBLIC MEDIA

  - "Plotting trees + data" lecture by Guangchuang Yu.
https://youtu.be/tqspAXM0UQE
<!-- or https://v.qq.com/x/page/k0565r0qiyy.html -->


# \faHome \hspace{2pt} PROFESSIONAL AFFILIATIONS
  - **Member**, International Society for influenza and other Respiratory Virus
    Disease \hfill 2016
  - **Steering Committee**, International Network for Data Analysis, Institut
    Pasteur \hfill 2015
  - **Visiting student**, Beijing Institute of Radiation Medicine \hfill 2005-2009

# \faUser \hspace{2pt} PROFESSIONAL SERVICES

## **Associate Editor**
  - *Journal of Integrated OMICS*, 2011-present

## **Reviewer**

> *Bioinformatics*, *BMC Bioinformatics*, *Computational Biology and Chemistry*,
> *Current Bioinformatics*, *Gastroenterology Research and Practice*, *Molecular
> BioSystems*, *R Journal*



# \faGears \hspace{2pt} SOFTWARE DEVELOPMENT

```{r echo=F, message=F, fig.asp=0.5}
packageTitle = ypages:::packageTitle
```

> **\textsf{R} ranking**: ranked `r gitaward("guangchuangyu", "r")` worldwide by [Git Awards](http://git-awards.com/users/guangchuangyu)

+ [badger](https://cran.r-project.org/package=badger):   `r packageTitle('badger', 'CRAN')`
+ [dlstats](https://cran.r-project.org/package=dlstats):   `r packageTitle('dlstats', 'CRAN')`
+ [emojifont](https://guangchuangyu.github.io/emojifont):   `r packageTitle('emojifont', 'CRAN')`
+ [ggimage](https://cran.r-project.org/package=ggimage):   `r packageTitle('ggimage', 'CRAN')`
+ [meme](http://cran.r-project.org/package=meme):   `r packageTitle('meme', 'CRAN')`
+ [rvcheck](https://cran.r-project.org/package=rvcheck):   `r packageTitle('rvcheck', 'CRAN')`
+ [scatterpie](https://cran.r-project.org/package=scatterpie):   `r packageTitle('scatterpie', 'CRAN')`
+ [shadowtext](http://cran.r-project.org/package=shadowtext):   `r packageTitle('shadowtext', 'CRAN')`
+ [ChIPseeker](https://guangchuangyu.github.io/ChIPseeker):   `r packageTitle('ChIPseeker', 'BioC')`
+ [clusterProfiler](https://guangchuangyu.github.io/clusterProfiler):   `r packageTitle('clusterProfiler', 'BioC')`
+ [DOSE](https://guangchuangyu.github.io/DOSE):   `r packageTitle('DOSE', 'BioC')`
+ [ggtree](https://guangchuangyu.github.io/ggtree):   `r packageTitle('ggtree', 'BioC')`
+ [GOSemSim](https://guangchuangyu.github.io/GOSemSim):   `r packageTitle('GOSemSim', 'BioC')`
+ [meshes](https://guangchuangyu.github.io/meshes):   `r packageTitle('meshes', 'BioC')`
+ [ReactomePA](https://guangchuangyu.github.io/ReactomePA):   `r packageTitle('ReactomePA', 'BioC')`
+ [seqcombo](https://www.bioconductor.org/packages/seqcombo):   `r packageTitle('seqcombo', 'BioC')`
+ [tidytree](https://cran.r-project.org/package=tidytree):   `r packageTitle('tidytree', 'CRAN')`
+ [TIPars](https://github.com/id-bioinfo/TIPars):   Taxa Insertion by Parsimony
+ [treeio](https://bioconductor.org/packages/treeio):   `r packageTitle('treeio', 'BioC')`

## **Contributions to open source libraries**

+ [basicTrendline](https://cran.r-project.org/package=basicTrendline):   `r packageTitle('basicTrendline', 'CRAN')`
+ [cowsay](https://cran.r-project.org/package=cowsay):   `r packageTitle('cowsay', 'CRAN')`
+ [kableExtra](https://cran.r-project.org/package=kableExtra):   `r packageTitle('kableExtra', 'CRAN')`
+ [MSnbase](https://bioconductor.org/packages/MSnbase/):   `r packageTitle('MSnbase', 'BioC')`
+ [scholar](https://github.com/jkeirstead/scholar):   `r packageTitle('scholar', 'CRAN')`

